ferveo/src/pvss.rs

```diff
@@ -150,7 +156,7 @@ impl<E: Pairing, T> PubliclyVerifiableSS<E, T> {
                 // ek_{i}^{eval_i}, i = validator index
                 fast_multiexp(
                     // &evals.evals[i..i] = &evals.evals[i]
-                    &[evals.evals[validator.share_index as usize]], // one share per validator
+                    &[evals[validator.share_index as usize]], // one share per validator
```

Comment:
Not necessarily for this PR, but we should get rid of this one-element multiexp and just make a normal one.